src/components/map/components/action/layer-action/select-action/index.js

```diff
@@ -72,10 +72,23 @@ export class VlMapSelectAction extends VlMapLayerAction {
     }
   }
 
+  /**
```

Comment:
`vl-mapactions/dist/vl-mapactions.js` import should occur before import of `../../../../../../utils/core`

Reply:
Ik pas het aan (zoals de rest), maar hoort de linting dit soort dingen niet tijdens de build te vangen?

Reply:
Ik vind het best raar dat je editor onze eslint config negeert (blijkbaar). Anyway we zouden inderdaad eens moeten kijken naar bv husky ofzo, voor wat precommit rules.

Reply:
Da's waarschijnlijk gewoon omdat ik op een nieuwe laptop bezig ben en hem nog niet goed had ingesteld. Webstorm pakt dat normaal inderdaad gewoon mee.

Reply:
Maar moet wel mee in de build zitten in plaats van in ieders IDE uiteraard.

Reply:
Yes, je kan in principe gewoon eslint aanroepen vanuit een van de package.json tasks, bij het builden. Dat was inderdaad wat ik initieel bedoelde.
Dan is het nog niet precommit, maar dan vang je het wel al op.